pytest_xvfb.py

```diff
@@ -130,8 +132,13 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption('--no-xvfb') or not xvfb_available():
+    if config.getoption('--no-xvfb') or not sys.platform.startswith('linux'):
         config.xvfb = None
+    elif (not xvfb_available() and sys.platform.startswith('linux') and
+          os.environ.get('DISPLAY') is not None):
+        config.xvfb = None
+        warnings.warn(
+            'pytest-xvfb could not find Xvfb. You can install it to avoid windows from opening.')
```

Comment:
Thanks, I'll membership check for DISPLAY in os.environ. I prefer to avoid nested if statements in code, however I can change the code block if you think it's easier to read.

As far as I'm aware, pytest-warnings needs the pytest-warnings plugin in order for the user to see a warnings report. If this is not the case, please let me know.